flow-report/test/run-flow-report-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+##
+# @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+##
+
+set -eux
+
+ARGS=(
+  --testMatch='{flow-report/**/*-test.ts,flow-report/**/*-test.tsx}'
+  --require=flow-report/test/setup/env-setup.ts
+  --loader=@esbuild-kit/esm-loader
+  # util-test.tsx won't finish on its own because of an open MessagePort, so help it out.
+  # See https://github.com/jsdom/jsdom/issues/2448#issuecomment-802288244
+  --exit
+)
+
+yarn mocha ${ARGS[*]} "$@"
```

Comment:
Would be nice if we could run any test from a single command. WDYT about conditionally adding these flags in `run-tests.js`.

Reply:
what do you mean? can just do `yarn unit-flow <test file>`

Reply:
Oh, I see. This keeps it simpler, since only these tests need to user different loaders or requires. To do that in the other file would mean partitioning tests that need different contexts.

Reply:
> To do that in the other file would mean partitioning tests that need different contexts.

Yeah that's what I was suggesting. I guess it was nice when jest let you define a new test environment in `jest.config.js`.